From dbfc08b9dc879e21bfacbd153948d21db36d50cd Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Thu, 23 Nov 2023 13:20:40 +0100
Subject: [PATCH] meson.build: set hardcoded version string

Signed-off-by: Markus Volk <f_l_k@t-online.de>

This fixes version string in the pkgconfig file and thus unbreaks spice build
| spice-common| Run-time dependency libcacard found: NO
| 
| ../git/subprojects/spice-common/meson.build:144:16: ERROR: Dependency lookup for libcacard with method 'pkgconfig' failed: Invalid version, need 'libcacard' ['>= 2.5.1'] found 'UNKNOWN'.
| 
| A full log can be found at /home/flk/poky/build/tmp/work/corei7-64-poky-linux/spice/0.15.2/build/meson-logs/meson-log.txt
| ERROR: meson failed

Upstream-Status: Inappropriate [oe-specific]
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 9cfe248..00745e5 100644
--- a/meson.build
+++ b/meson.build
@@ -1,5 +1,5 @@
 project('libcacard', 'c', 'cpp',
-  version: run_command('build-aux/git-version-gen', '@0@/.tarball-version'.format(meson.source_root()), check: true).stdout().strip(),
+  version: '2.8.1',
   license: 'LGPLv2.1',
   default_options: ['warning_level=2'],
 )
-- 
2.42.0

